mdx/guides/0x-api-faq.mdx

```diff
@@ -0,0 +1,142 @@
+
+**TLDR** `slippagePercentage` controls how much worse the price can be for the fallback orders provided in a 0x API quote which influences the `guaranteedPrice`.
+
+## What are the differences when quoting by `sellAmount` and `buyAmount`?
+
+As alluded earlier, the amount of `buyToken` received is not strictly deterministic with a quote, a number of factors influence the actual settled amount of `buyTokens` received. 
+
+Consider if you receive a better price and not all `sellToken` is utilized:
+
+* If `sellToken` is utilized, then any unused `sellToken` will be refunded to the user.
+* When `buyAmount` is used, the only guarantee is that **at least** the amount specified is bought. 0xAPI will not terminate early in the case where one order fills at a better price, so the user can in effect over buy the specified amount. This is somewhat amplified by usage of `slippagePercentage` which underestimates the on-chain price by a percentage.
+
+
+Also, some liquidity sources do not enable querying by `buyAmount` (i.e Kyber), these sources are ignored when quoting for `buyAmounts`
+
+**TLDR** Whenever possible, use `sellAmount` over `buyAmount` to get more deterministic behavior.
+
+## How does 0x API select the best orders for me?
+
+Beyond simply sampling each liquidity source for their respective prices, 0x API adjusts for the gas consumption of each liquidity source with the specified gas price (if none provided 0x API will use ethGasStation's `fast` amount of gwei) and any associated fees with the specific liquidity source. By sampling through varying compositions of liquidity sources, 0x API selects the best set of orders to give you the best price. 0x API also creates another set of fallback orders to ensure that the quote can be executed by users.
+
+Ex: 0x API will adjust the price potentially received from Curve Finance by gas * gasPrice and its fees. Because of Curve Finance’s costly gas consumption, its nominal price may not be the best price when settled.
+
+## What is the `protocolFee`? 
+
+A protocol fee is paid by takers and ultimately rebated to market makers when their orders are filled. Protocol fees are calculated per order using the gas price multiplied by a constant (currently 150,000). 
+
+0xAPI calculates the required protocol fees to be paid and returns this in the `value` field. Since we recommend using a high gas price, in the event where the taker fills at a lower gas price, the excess is returned. 
```

Comment:
```suggestion
0x API calculates the required protocol fees to be paid and returns this in the `value` field. Since we recommend using a high gas price, in the event where the taker fills at a lower gas price, the excess is returned. 
```